Dockerfile

```diff
@@ -1,75 +1,27 @@
-# BSD License
-#
-# Copyright (C) 2017-2019 Baidu, Inc. All Rights Reserved.
-#
-# Redistribution and use in source and binary forms, with or without
-# modification, are permitted provided that the following conditions
-# are met:
-#
-#   * Redistributions of source code must retain the above copyright
-#     notice, this list of conditions and the following disclaimer.
-#   * Redistributions in binary form must reproduce the above copyright
-#     notice, this list of conditions and the following disclaimer in
-#     the documentation and/or other materials provided with the
-#     distribution.
-#   * Neither the name of Baidu, Inc., nor the names of its
-#     contributors may be used to endorse or promote products derived
-#     from this software without specific prior written permission.
-#
-# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
-# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
-# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
-# A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
-# OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
-# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
-# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
-# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
-# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-#
-# Modifications Copyright 2019 Foris Limited (licensed under the Apache License, Version 2.0)
+FROM baiduxlab/sgx-rust:1604-1.0.8
```

Comment:
can it be based on 1804?

Reply:
Let me try, it should still work

Reply:
make 

*** No rule to make target `/opt/intel/sgxsdk/bin/x64/sgx_edger8r', needed by `enclave/Enclave_t.c'.  Stop

Reply:
os is mac

Reply:
Changed to use 18.04 image @tomtau

Reply:
`/opt/intel/sgxsdk/bin/x64/sgx_edger8r`, the path prefix should not be `/opt/intel/sgxsdk`. Perhaps the `~/.docker_bashrc` is not sourced properly. We can investigate on your Mac tomorrow.

Reply:
ok